src/views/ViewTrackDetails.tsx

```diff
@@ -115,6 +133,130 @@ export default function ViewTrackDetails() {
             }}
           />
         </Setting.Section>
+        <Setting.Section>
+          <Setting.Input
+            label="Year"
+            id="year"
+            name="year"
+            type="number"
+            min="1900"
+            step="1"
+            value={formData.year}
+            onChange={(e) => {
+              setFormData({ ...formData, year: Number(e.currentTarget.value) });
+            }}
+          />
+        </Setting.Section>
+        <Setting.Section>
+          <Setting.Input
+            label="Duration"
+            id="duration"
+            name="duration"
+            type="text"
+            pattern="([01]?[0-9]{1}|2[0-3]{1}):[0-5]{1}[0-9]{1}"
+            readOnly={true}
+            value={formatDuration(formData.duration)}
+            onChange={(e) => {
+              setFormData({
+                ...formData,
+                duration: Number(e.currentTarget.value),
+              });
+            }}
+          />
+        </Setting.Section>
+        <Setting.Section>
+          <Flexbox direction="horizontal" gap={16}>
+            <Setting.Input
+              label="Track Nº"
+              id="track"
+              name="track"
+              type="number"
+              min="0"
+              step="1"
+              value={Number(formData.track.no)}
+              onChange={(e) => {
+                setFormData({
+                  ...formData,
+                  track: {
+                    no: Number(e.currentTarget.value),
```

Comment:
Same for all these, they should allow empty values (that set the value to null), not 0 by default.